app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -71,7 +71,13 @@ def create
 
     def destroy
       if @admin_set.destroy
-        redirect_to hyrax.admin_admin_sets_path, notice: t(:'hyrax.admin.admin_sets.delete.notification')
+        if request.referer.include? "my/collections"
```

Comment:
Please move this into a method called `after_delete_success`. That way it's easy to override the single method if these paths need to be changed in a local application.